### ISSUE TITLE
How to compute training loss?

### ISSUE THREAD

**Issue by huiminzeng** [MIC-DKFZ/ACDC2017#2]
Hi!

Thank you for the code! But may I ask how the training loss is computed? I mean the ground truth exist for only 2 time steps, right? Is it true that the models were only trained with data at those 2 time steps?

thx

**Comment by FabianIsensee** [comment]
Hi,
we treat the time steps as independent training examples. The ground truth is only available for these two, so yes we train only on them. With 100 cases in the training set and 2 time steps per case that is 200 training images in total
Best,
Fabian

**Comment by huiminzeng** [comment]
i see. thx
